src/m365/spo/commands/site/site-set.ts

```diff
@@ -670,31 +670,16 @@ class SpoSiteSetCommand extends SpoCommand {
     return Cli.executeCommand(spoSiteDesignApplyCommand as Command, { options: { ...options, _: [] } });
   }
 
-  private loadSiteIds(siteUrl: string, logger: Logger): Promise<void> {
+  private async loadSiteIds(adminUrl: string, siteUrl: string, logger: Logger): Promise<void> {
```

Comment:
The `adminUrl` is a class property, so let's just use it and not add it as a parameter